### PR TITLE
Build support for Precise and Trusty

### PR DESCRIPTION
**Pull request by daniviga**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,7 @@
+  [Matteo Nastasi]
```
Comment (nastasi-oq) [inline debian/changelog]: No item with your job.

File: packager.sh [hunk 2]
```diff
@@ -709,7 +753,7 @@ if [ $BUILD_DEVEL -eq 1 ]; then
         pkg_deb="-0"
```
Comment (nastasi-oq) [inline packager.sh]: at line 735 could we uncomment echo "pkg [$pkg_vers] [$pkg_maj] [$pkg_min] [$pkg_bfx] [$pkg_deb] [$pkg_suf]" to be consistent with 'hazardlib' if we decide to keep it uncommented

Comment (daniviga) [inline packager.sh]: Line is now commented out in oq-hazardlib

**Comment by daniviga** [comment]
Done

**Comment by daniviga** [comment]
LGTM, waiting for @nastasi-oq LGTM

**Comment by nastasi-oq** [comment]
LGTM

**Pull request merged**